pyccl/ccl_sigM.i

```diff
@@ -0,0 +1,101 @@
+%module ccl_massfunc
+
+%{
+/* put additional #include here */
+%}
+
+%include "../include/ccl_massfunc.h"
+
+// Enable vectorised arguments for arrays
+%apply (double* IN_ARRAY1, int DIM1) {(double* logM, int nM)};
+%apply (int DIM1, double* ARGOUT_ARRAY1) {(int nout, double* output)};
+
+/* The python code here will be executed before all of the functions that
+   follow this directive. */
+%feature("pythonprepend") %{
+    if numpy.shape(logM) != (nout,):
+        raise CCLError("Input shape for `halo_mass` must match `(nout,)`!")
+%}
+
+%inline %{
+
+void sigM_vec(ccl_cosmology * cosmo, double a,
+	      double *logM, int nM,
+	      int nout, double* output, int *status)
+{
+  if (cosmo->params.N_nu_mass>0){
+	  *status = CCL_ERROR_NOT_IMPLEMENTED;
+	  ccl_cosmology_set_status_message(cosmo, "ccl_background.c: ccl_cosmology_compute_growth(): Support for the sigma(M) function in cosmologies with massive neutrinos is not yet implemented.\n");
+	  return;
+  }
+
+  // Check if sigma has already been calculated
+  if (!cosmo->computed_sigma) {
+    *status = CCL_ERROR_SIGMA_INIT;
+    ccl_cosmology_set_status_message(
+      cosmo,
+      "ccl_massfunc.c: ccl_sigmaM(): linear power spctrum has not been computed!");
```

Comment:
Typo.

Reply:
removed this (this is already checked at the python level)

Reply:
You marked this as removed but I still see the code. Did you forget to push?